bitcoind-rpc/src/main/scala/org/bitcoins/rpc/client/v18/BitcoindV18RpcClient.scala

```diff
@@ -0,0 +1,60 @@
+package org.bitcoins.rpc.client.v18
+import akka.actor.ActorSystem
+import org.bitcoins.rpc.client.common.{BitcoindRpcClient, BitcoindVersion, Client}
+import org.bitcoins.rpc.config.BitcoindInstance
+import org.bitcoins.rpc.jsonmodels.{GetNodeAddressesResult, GetRpcInfoResult, ListWalletDirResult, SubmitHeaderResult}
+import org.bitcoins.rpc.serializers.JsonSerializers._
+import play.api.libs.json.{JsString, Json}
+
+import scala.concurrent.Future
+import scala.util.Try
+
+class BitcoindV18RpcClient(override val instance: BitcoindInstance)(
+    implicit
+    actorSystem: ActorSystem)
+    extends BitcoindRpcClient(instance)
+    with V18PsbtRpc
+    with V18DescriptorRpc
+    with V18AssortedRpc {
+
+  override lazy val version: BitcoindVersion = BitcoindVersion.V18
+
+}
+
+object BitcoindV18RpcClient {
+
+  def fromUnknownVersion(rpcClient: BitcoindRpcClient)(
+      implicit actorSystem: ActorSystem): Try[BitcoindV18RpcClient] =
+    Try {
+      new BitcoindV18RpcClient(rpcClient.instance)
+    }
+
+}
+
+trait V18AssortedRpc {
```

Comment:
I think having this in a separate file would be good, also a simple scaladoc would be nice